### ISSUE TITLE
GOG/STEAM

### ISSUE THREAD

**Issue by craxton** [Nukem9/dlssg-to-fsr3#256]
tried on Hogwarts Legacy via Epic games store version (latest drivers etc) win 11, And Cyberpunk 2077 with PL DLC, on GOG version. Neither works nothing i can do gets those up and going. However, Steams version of Starfield works without issues. 

Im unsure if this is by design, or i have some underlying other issue. But i did exactly what the directions stated and copied the files to phoenix/binaries/x64 to no avail as well as the install location for CP2077. again, starfield worked without issues

5800x, 3070ti, latest drivers/version of win 11.

**Comment by Or0b0r0s** [comment]
Very strange thing for me too some games works and others none 🤷‍♂️

**Comment by craxton** [comment]
its strange bc the ones im trying are "verified" on the working list of this mod. So, im pretty lost on whats wrong. Im honestly leaning more towards the fact epic games sucks, and IDK whats going on with GOG, have reports GOG version of CP2077+PL dlc works, but has graphical issues. never the less, its still the point that this isnt a "hardware" limitation and is only software locked. To think i was just about ready to snag a 4070ti (even tho) i just purchased this 3070ti ftw3 like 5 weeks ago...

**Comment by nefh** [comment]
> Cyberpunk 2077 with PL DLC, on GOG version. Neither works

@craxton something is wrong on your side. I am using it with CP2077+PL on GOG just fine. Have you pasted .dlls in correct directory? (\Program Files\Cyberpunk 2077\bin\x64)

**Comment by craxton** [comment]
No bud, its lack of "removing" something in GOG as i indeed got it up and going on Starfield (steam version) and Hogwarts legacy (epic games version) (deleted portal folder contents) but GOG on cp2077 refuses to allow FG. The files are 100% in the right spot. (i do mod, use cracked games etc) so replacing "modded" files is well within my abilities lol

note: all games im trying this on, are paid for, so far GOG has been the only issue. Im sure ill get around it eventually.

**Comment by nefh** [comment]
> to also add, you cant really mess it up if you have GOG open and go thru GOG to get to where you have "X" game installed haha...

Yeah, but GOG is leading exactly to wrong directory, with wrong .dlls (`C:\Program Files\Cyberpunk 2077` instead of `C:\Program Files\Cyberpunk 2077\bin\x64`)

Anyway, this is not general problem, since I have GOG version of CP2077+PL running just fine with framegen.

**Comment by craxton** [comment]
your arguments not valid considering GOG opens my game up to the correct folder "YET AGAIN" ill state that. ill go as far as showing you i suppose. and give it one last shot. now,  if it works ill remove this entire thread, considering it wasnt up-to date earlier which may had been a factor. however, considering other games (hogwarts although crashes even tho its not the "pirated jack sparrow" version) and starfield. so hopeful the update was the key issue. 

![Screenshot (102)](https://github.com/Nukem9/dlssg-to-fsr3/assets/62484595/6cde4a2c-30e0-4218-9d74-f6a591bfc25f)
![Screenshot (103)](https://github.com/Nukem9/dlssg-to-fsr3/assets/62484595/ade27ab2-4690-4a7f-91f0-1105d72d20c8)
